onnx/defs/schema.cc

```diff
@@ -861,13 +866,15 @@ OpName_Domain_Version_Schema_Map& OpSchemaRegistry::map() {
 
 #ifndef NDEBUG
       size_t dbg_registered_schema_count = GetRegisteredSchemaCount() - dbg_initial_schema_count;
-
-      ONNX_ASSERTM(
-          dbg_registered_schema_count == ONNX_DBG_GET_COUNT_IN_OPSETS(),
-          "%u schema were exposed from operator sets and automatically placed into the static registry.  "
-          "%u were expected based on calls to registration macros. Operator set functions may need to be updated.",
-          dbg_registered_schema_count,
-          ONNX_DBG_GET_COUNT_IN_OPSETS());
+      // Don't check if the schema was partially loaded
+      if (OpSchemaRegistry::Instance()->GetLoadedSchemaVersion() > 0) {
```

Comment:
did you mean if loaded_schema_version == 0? in operator_sets.h you set this to 0 when you are loading all schemas and that is when you want to run this check right

Reply:
Good catch! Should be ```if loaded_schema_version == 0``` here. Will update. Thanks